python/felis/datamodel.py

```diff
@@ -462,3 +465,27 @@ def require_description(cls, rd: bool = True) -> None:
     def is_description_required(cls) -> bool:
         """Return whether a description is required for all objects."""
         return cls.ValidationConfig._require_description
+
+    @classmethod
+    def from_yaml_path(cls, yaml_path: str) -> Schema:
+        with open(yaml_path, "r") as yaml_file:
```

Comment:
Can't this method call `from_yaml_file` and so avoid the code duplication?

Reply:
see https://github.com/lsst/felis/pull/47#discussion_r1525442939